### PR TITLE
git-test 1.0.1 (new formula)

### PR DESCRIPTION
**Pull request by wjlroe**
From the [README](https://github.com/spotify/git-test/blob/master/README.md):

> Run tests on each distinct tree in a revision list, skipping versions whose contents have already been tested.

### CODE REVIEW

File: Library/Formula/git-test.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  def install
+    prefix.install "test.sh"
+    bin.install "git-test"
+    man1.install "git-test.1"
+  end
+
+  test do
+    system "git", "init"
+    ln_s "#{bin}/git-test", getwd
+    cp File.join(prefix, "test.sh"), getwd
```
Comment (mikemcquaid) [inline Library/Formula/git-test.rb]: `cp "#{prefix}/test.sh", testpath`

File: Library/Formula/git-test.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+  def install
+    prefix.install "test.sh"
+    bin.install "git-test"
+    man1.install "git-test.1"
+  end
+
+  test do
+    system "git", "init"
+    ln_s "#{bin}/git-test", getwd
```
Comment (mikemcquaid) [inline Library/Formula/git-test.rb]: Use `testpath` instead of `getwd`

File: Library/Formula/git-test.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
+class GitTest < Formula
+  homepage "https://github.com/spotify/git-test"
+  url "https://github.com/spotify/git-test/archive/v1.0.1.tar.gz"
+  sha256 "1273d97644edf3690cc0310a1f6f33856f83bf736599c54150db7a7ad9e43983"
+
+  def install
+    prefix.install "test.sh"
```
Comment (mikemcquaid) [inline Library/Formula/git-test.rb]: Put this under `share` somewhere I think

File: Library/Formula/git-test.rb [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  end
+
+  test do
+    system "git", "init"
+    ln_s "#{bin}/git-test", getwd
+    cp File.join(prefix, "test.sh"), getwd
+    chmod 0755, "test.sh"
+    system "git", "add", "test.sh"
+    system "git", "commit", "-m", "initial commit"
+    system "#{bin}/git-test", "-v", "HEAD", "--verify='./test.sh'"
```
Comment (mikemcquaid) [inline Library/Formula/git-test.rb]: Need to set `ENV["TERM"]` to something (e.g. `xterm`): http://bot.brew.sh/job/Homebrew%20Pull%20Requests/23669/version=yosemite/testReport/junit/brew-test-bot/yosemite/test_git_test/

**Comment by wjlroe** [comment]
Oh, I didn't realise that if I amended the commit and force pushed then Jenkins wouldn't build the amended commit.

**Comment by wjlroe** [comment]
Never mind, the build worked. All comments have been implemented I think. Thanks!

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**